addons/binding/pom.xml

```diff
@@ -50,6 +50,7 @@
     <module>org.openhab.binding.dsmr</module>
     <module>org.openhab.binding.dsmr.test</module>
     <module>org.openhab.binding.elerotransmitterstick</module>
+    <module>org.openhab.binding.elkm1</module>
```

Comment:
Please also add a feature for this binding to:

`features/openhab-addons/src/main/feature/feature.xml`